config/initializers/secrets.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# Be sure to restart your server after modifying this file.
+
+require "pathname"
+require "velum/secrets"
+
+env_path = Pathname.new("#{ENV.fetch("VELUM_SECRETS_DIR")}/key_base.json")
```

Comment:
i think a `pathname` object makes more sense here, as it is a handy preparation for all kinds of file operations